versions/weekly/flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  inputs =
+    {
+      holochain = {
+        url = "github:holochain/holochain/holochain-0.3.0-beta-dev.8";
+        flake = false;
+      };
+
+      lair = {
+        url = "github:holochain/lair/lair_keystore-v0.2.4";
+        flake = false;
+      };
+
+      # holochain_cli_launch
+      launcher = {
+        url = "github:holochain/launcher/holochain-0.2";
```

Comment:
obviously this branch of the launcher is targeting the holochain 0.2 series, so there's likely incompatibility; not by intention but most likely by happenstance

Reply:
I agree, and per the comment above it would make sense to add at least smoke tests to check these versions work together. I think we're overdue on needing that but right now I really want to focus on getting the 0.2.1 version released. I think this should be a high priority change following that release